release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1080,6 +1080,78 @@ With this release, the CCO no longer checks for the nonexistent permission.
+
+* Previously, installations on {gcp-short} could fail because the installation program attempted to create a bucket twice. With this release, the installation program no longer attempts to create the bucket twice. (link:https://issues.redhat.com/browse/OCPBUGS-32133[*OCPBUGS-32133*])
+
+* Previously, a rare timing issue could prevent all control plane nodes from being added to an Agent-based cluster during installation. With this release, all control plane nodes are successfully rebooted and added to the cluster during installation. (link:https://issues.redhat.com/browse/OCPBUGS-32105[*OCPBUGS-32105*])
+
+* Previously, when using the Agent-based installation program in a disconnected environment, unnecessary certificates were added to the CA trust bundle. With this release, the CA bundle `ConfigMap` only contains CAs explicitly specified by the user. (link:https://issues.redhat.com/browse/OCPBUGS-32042[*OCPBUGS-32042*])
+
+* Previously, the installation program required a non-existent permission `s3:HeadBucket` when installing a cluster on AWS. With this release, the installation program correctly requires the permission `s3:ListBucket` instead. (link:https://issues.redhat.com/browse/OCPBUGS-31813[*OCPBUGS-31813*])
+
+* Previously, if the installation program failed to gather logs from the bootstrap due to an SSH connection issue, it would also not provide virtual machine (VM) serial console logs even if they were collected. With this release, the installation program provides VM serial console logs even if the SSH connection to the bootstrap machine fails. (link:https://issues.redhat.com/browse/OCPBUGS-30774[*OCPBUGS-30774*])
+
+* Previously, when installing a cluster on {vmw-full} with static IP addresses, the cluster could create control plane machines without static IP addresses due to a conflict with other Technology Preview features. With this release, the Control Plane Machine Set Operator correctly manages the static IP assignment for control plane machines. (link:https://issues.redhat.com/browse/OCPBUGS-29114[*OCPBUGS-29114*])
+
+* Previously, when installing a cluster on {gcp-short} with user-provided DNS, the installation program still attempted to validate DNS within the {gcp-short} DNS network. With this release, the installation program does not perform this validation for user-provided DNS. (link:https://issues.redhat.com/browse/OCPBUGS-29068[*OCPBUGS-29068*])
+
+* Previously, when deleting a private cluster on {ibm-cloud-name} that used the same domain name as a non-private {ibm-cloud-name} cluster, some resources were not deleted. With this release, all private cluster resources are deleted when the cluster is destroyed. (link:https://issues.redhat.com/browse/OCPBUGS-28870[*OCPBUGS-28870*])
```

Comment:
There might be an alternate word for 'destroy', per the ISG. Perhaps `remove` or 'when the cluster is taken down`?
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d

Reply:
:heavy_check_mark: . Went with "removed".